components/button/button.md

```diff
@@ -27,6 +27,9 @@ Variations - Ghost:
       <Button onClick={alert} variation="ghost" size="m">Medium Ghost</Button><br/><br/>
       <Button onClick={alert} variation="ghost" size="l">Large Ghost</Button><br/><br/>
       <Button onClick={alert} variation="ghost" size="xl">Extra large Ghost</Button><br/><br/>
+      <div style={{background: "#005FBB", padding: "10px", display: "inline-block"}}>
```

Comment:
what is the purpose to hard code the styles here?

Reply:
@iwwwi this is just for documentation. 

![selection_123](https://user-images.githubusercontent.com/10462372/31764953-75b0f5ac-b4cb-11e7-9166-6a6eff226863.png)

Reply:
maybe because those styles are not related to the button components. It's background under a button which shows how button will look like on a dark background @froskie what do you think? Should I move it to sass file?

Reply:
@AlleeX @MadinaShad cool :) doesn't have to remove it ;)